vstsBuild/vstsBuild.psm1

```diff
@@ -11,6 +11,43 @@ function Clear-VstsTaskState
     $script:taskstate = $succeededStateName
 }
 
+function Get-TempFolder
```

Comment:
It's a duplicate of a function with the same name in [dockerBasedBuild.common.psm1](https://github.com/PowerShell/PSRelease/blob/693039b63d9f9c2a48de4ca0e6d1535d3b583a24/dockerBasedBuild/dockerBasedBuild.common.psm1#L35). Possible to reuse the same one?

Reply:
I filed an issue. https://github.com/PowerShell/PSRelease/issues/21